fhir-server-test/src/test/java/com/ibm/fhir/server/test/SystemHistoryTest.java

```diff
@@ -464,10 +465,36 @@ public void testSystemHistoryWithTypePatientAndOrderDESC() throws Exception {
                 assertNotNull(nextPath);
             }
         } while (count > 0);
-        
+
         assertTrue(found, "Patient id in history");
     }
 
+    @Test
+    public void testSystemHistoryWithNoType_tenant1() throws Exception {
+        WebTarget target = getWebTarget();
```

Comment:
Since this test is similar to SearchAllTest:testSearchAllUsingId_tenant1, I think adding a similar comment like "tenant1 is configured to support history on only a subset of resource types" would make it clear why the `_type` parameter should exist in the self link.

Reply:
done